pact/Pact/Core/IR/Eval/CoreBuiltin.hs

```diff
@@ -1620,11 +1697,13 @@ coreDefineNamespace info b cont handler env = \case
     let pdb = view cePactDb env
     let nsn = NamespaceName n
         ns = Namespace nsn usrG adminG
+    chargeGasArgs info $ GRead $ sizeOf SizeOfV0 n
```

Comment:
Ditto here, why do we charge `GRead` on `n` when it's just a string?

On `existing` it's fine.